test/dummy/db/migrate/20150429090540_create_suscribir_suscripciones.suscribir.rb

```diff
@@ -0,0 +1,24 @@
+# This migration comes from suscribir (originally 20140219153752)
+class CreateSuscribirSuscripciones < ActiveRecord::Migration
+  def change
+    create_table :suscribir_suscripciones, force: true do |t|
+      t.integer  :suscribible_id, null: false
+      t.integer  :suscribible_type, null: false
+      t.string   :dominio_de_alta, null: false, default: 'es'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.